protocol/message.go

```diff
@@ -46,5 +46,16 @@ type KeyLookupRequest struct {
 type KeyLookupProof struct {
 	Type int
 	AP   *merkletree.AuthenticationPath
-	TB   *merkletree.TemporaryBinding
+	TB   *merkletree.TemporaryBinding `json:",omitempty"`
+}
+
+type ConsistencyCheckRequest struct {
+	Username string `json:"username"`
+	Epoch    uint64 `json:"epoch"`
+}
+
+type ConsistencyCheckProof struct {
```

Comment:
With the STR added to `KeyLookupProof`, this essentially becomes the same struct (except without the optional TB field). I think we could merge these structs, but keep the `ConsistencyCheckProof` name since I think it reflects better how clients operate on this message.